parts/k8s/containeraddons/1.17/kubernetesmasteraddons-cluster-autoscaler-deployment.yaml

```diff
@@ -53,15 +53,19 @@ rules:
 - apiGroups: ["apps"]
   resources: ["statefulsets","replicasets","daemonsets"]
   verbs: ["watch","list","get"]
-- apiGroups: ["batch"]
-  resources: ["jobs"]
-  verbs: ["watch","list"]
 - apiGroups: ["storage.k8s.io"]
-  resources: ["csinodes", "storageclasses"]
+  resources: ["storageclasses", "csinodes"]
```

Comment:
Order wasn't significant here, right?

Reply:
No it's not. I'm just trying to minimize the diff between our manifest and the upstream master example.